includes/class-freemius.php

```diff
@@ -11365,6 +11378,40 @@ function get_trial_url() {
             return $this->get_upgrade_url( WP_FS__PERIOD_ANNUALLY, true );
         }
 
+        /**
+         * @author Leo Fajardo (@leorw)
+         * @since 2.1.4
+         *
+         * @param string $new_version
+         *
+         * @return string
+         */
+        function version_upgrade_checkout_link( $new_version ) {
+            if ( ! is_object( $this->_license ) ) {
+                $url = $this->pricing_url();
+
+                $purchase_license_text = $this->get_text_inline( 'Buy a license now', 'buy-license-now' );
+            } else {
+                $subscription = $this->_get_subscription( $this->_license->id );
+
+                $url = $this->checkout_url(
+                    is_object( $subscription ) ?
+                        ( 1 == $subscription->billing_cycle ? WP_FS__PERIOD_MONTHLY : WP_FS__PERIOD_ANNUALLY ) :
+                        WP_FS__PERIOD_LIFETIME,
+                    false,
+                    array( 'licenses' => $this->_license->quota )
+                );
+
+                $purchase_license_text = $this->get_text_inline( 'Renew your license', 'renew-your-license' );
+            }
+
+            return sprintf(
+                $this->get_text_inline( '%s to access version %s security & feature updates, and support.', 'renew-license-now' ),

```

Comment:
@fajardoleo `'renew-license-now'` is now conflicting since it's also used as the string key here:
```
fs_text_inline( 'Renew your license now', 'renew-license-now', $slug )
```

I suggest renaming the key to `'x-for-updates-and-support'`.